python_ta/contracts/__init__.py

```diff
@@ -221,6 +225,12 @@ def new_setattr(self: klass, name: str, value: Any) -> None:
                     else:
                         super(klass, self).__delattr__(name)
                     raise AssertionError(str(e)) from None
+        else:
+            caller_klass = type(caller_self)
+            if "__mutated_instances__" in caller_klass.__dict__:
+                mutable_instances = caller_klass.__dict__["__mutated_instances__"]
```

Comment:
The variable should probably be called `mutated_instances`

Reply:
Thanks for the consistency check 👍